style.css

```diff
@@ -116,3 +123,260 @@ ul {
+  line-height: 24px;
+  color: #000;
+  margin-bottom: 36px;
+}
+
+.content-project1 .lan .lang {
+  display: flex;
+  flex-direction: row;
+  justify-content: left;
+  background-color: #ebebff;
+  border-radius: 8px;
+  padding: 4px 12px;
+  margin-right: 8px;
+  margin-top: 12px;
+  text-align: center;
+  gap: 8px;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+}
+
+.see-p .see-project {
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
+  padding: 16px;
+  gap: 12px;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  width: 140px;
+  height: 48px;
+  background: #fff;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  text-align: center;
+  color: #396df2;
+}
+
+.first-project .snapshot1 {
+  width: 295px;
+  height: 220px;
+  background: url("images/project2.png");
+  border-radius: 8px;
+}
+
+.first-project .snapshot2 {
+  width: 295px;
+  height: 220px;
+  background: url("images/project3.png");
+  border-radius: 8px;
+}
+
+.first-project .snapshot3 {
+  width: 295px;
+  height: 220px;
+  background: url("images/project4.png");
+  border-radius: 8px;
+}
+
+.about-me {
+  width: 375px;
+  height: 1263px;
+  background: #fff;
+  display: flex;
+  padding: 114px 24px;
+  flex-wrap: wrap;
+  margin: 0;
+}
+
+.about-me .about {
+  display: flex;
+  flex-wrap: wrap;
+  width: 327px;
+  height: 409px;
+}
+
+.about-me .about h2 {
+  width: 327px;
+  height: 101px;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+.about-me .about p {
+  width: 327px;
+  height: 144px;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.about-me .about .connect {
+  width: 117px;
+  height: 24px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+}
+
+.about-me .resume .resumee {
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
+  padding: 12px;
+  gap: 12px;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  width: 162px;
+  height: 48px;
+  background: #fff;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  text-align: center;
+  color: #396df2;
+}
+
+.sort {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  display: flex;
+  flex-direction: column;
+}
+
+.sort .langu {
+  list-style: none;
+  width: 303px;
+  height: 66px;
+  background: #f7f7f9;
+  border-radius: 8px;
+  margin-bottom: 12px;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  margin-left: 0;
+  margin-top: 12px;
+}
```

Comment:
- [ ] [REQUIRED] Good job so far! Please make the changes shown in the images below =>

![image](https://user-images.githubusercontent.com/63915024/176978601-5531877b-a3f4-4b7b-ba7a-f6bee5ba9923.png)